ICAFIX/functionhighpassandvariancenormalize.m

```diff
@@ -93,16 +95,19 @@
 %Apply VN and Save HP_VN TCS
 if dovol > 0
     cts=cts./repmat(Outcts.noise_unst_std,1,ctsT);
+	% Use '_vnts' (volume normalized time series) as the suffix for the volumetric VN'ed TCS
     if hp>=0
-        save_avw(reshape(cts,ctsX,ctsY,ctsZ,ctsT),[fmri '_hp' num2str(hp) '_vn.nii.gz'],'f',[1 1 1 1]); 
-        call_fsl(['fslcpgeom ' fmri '.nii.gz ' fmri '_hp' num2str(hp) '_vn.nii.gz -d']); 
+        save_avw(reshape(cts,ctsX,ctsY,ctsZ,ctsT),[fmri '_hp' num2str(hp) '_vnts.nii.gz'],'f',[1 1 1 1]); 
+        call_fsl(['fslcpgeom ' fmri '.nii.gz ' fmri '_hp' num2str(hp) '_vnts.nii.gz -d']); 
     else
-        save_avw(reshape(cts,ctsX,ctsY,ctsZ,ctsT),[fmri '_vnf.nii.gz'],'f',[1 1 1 1]); 
-        call_fsl(['fslmaths ' fmri '.nii.gz -sub ' fmri '.nii.gz -add ' fmri '_vnf.nii.gz ' fmri '_vnf.nii.gz']); 
-        %call_fsl(['fslcpgeom ' fmri '.nii.gz ' fmri '_vnf.nii.gz -d']); 
+        save_avw(reshape(cts,ctsX,ctsY,ctsZ,ctsT),[fmri '_vnts.nii.gz'],'f',[1 1 1 1]);
+        % Following use of 'fslmaths' rather than 'fslcpgeom' is a hack due to FSL 6.0.0 version of the latter not being able to handle very large files.
```

Comment:
This comment is wrong, it is fsl 5 that has a problem with large files.  The problem in fsl 6.0.0 is in not setting all the header fields it is supposed to.

Reply:
In that case, why aren't all the other instances of 'fslcpgeom' in that script problematic as well?

Reply:
Some of them probably were, but didn't get tested on as large of datasets.

Reply:
As for the fsl 6.0.0 bug, we have a patched version of fslcpgeom.

Reply:
Do you know/recall which specific header fields aren't getting set correctly?

Reply:
Specifically, it only looks at the reference file's qform, and sets only the target file's sform.  There is a patched version they sent us privately.